src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java

```diff
@@ -87,10 +91,13 @@ public SunburstRingSegment(final PhylogeneticTreeItem value, final int layer,
      *            the X coordinate of the center of the circle
      * @param centerY
      *            the Y coordinate of the center of the circle
+     * @param scale
+     *            the scaling factor
      * @return a semi-circle with the specified dimensions
      */
     private Shape createRing(final int layer, final double degreeStart,
-            final double degreeEnd, final double centerX, final double centerY) {
+            final double degreeEnd, final double centerX, final double centerY,
```

Comment:
If you refactor above function to use point2d for centerX and centerY, don't forget to use this here as well